glide.yaml

```diff
@@ -6,7 +6,7 @@ import:
   subpackages:
   - statsd
 - package: github.com/facebookgo/clock
-  version: 600d898af40aa09a7a93ecb9265d87b0504b6f03
+  version: master
```

Comment:
Might be preferable to have no pin at all.

That way consumers of tally aren't forced to pin to master and can pin to a hash if that's what they need.

Reply:
I'd be in preference of just removing the dependency altogether.  

To mock a clock is really simple and we've done it tons of times just with a 20 line or so implementation.  You rarely need the extra features a package like this offers.

Reply:
Another option is to actually check in the vendored code for this library. facebookgo/clock doesn't cross Tally's API boundaries so github.com/uber-go/tally/vendor/github.com/facebookgo/clock being different from github.com/facebookgo/clock shouldn't be an issue.

This is something ThriftRW Go already does with github.com/fatih/structtag: https://github.com/thriftrw/thriftrw-go/tree/dev/vendor/github.com/fatih/structtag